bigtable/integration_test.go

```diff
@@ -2250,16 +2250,47 @@ func TestIntegration_AdminBackup(t *testing.T) {
 
 	adminClient, err := testEnv.NewAdminClient()
 	if err != nil {
-		t.Fatalf("NewAdminClient: %v", err)
+		t.Errorf("NewAdminClient: %v", err)
```

Comment:
Same comment. If there was an error creating `adminClient`, it doesn't make sense to continue the test as `adminClient.CreateTableFromConf` will always fail.